src/serviceFacades/bags.js

```diff
@@ -171,40 +124,29 @@ export const useBagAddItem = (
 };
 
 export const useBagAddItems = (
-    { handleError, handleSuccess, handleSettled } = {
+    { handleError, handleSuccess } = {
         handleError: (error) => {
             console.log(`error from useBagAddItems: ${error.message}`);
         },
         handleSuccess: null,
-        handleSettled: null,
     }
 ) => {
-    const queryCache = useQueryCache();
-
+    const [bagInfo, setBagInfo] = useBagInfo();
     const successFn = (data, variables) => {
-        queryCache.setQueryData(DEFAULT_BAG_QUERY_KEY, data);
+        setBagInfo(data);
 
         if (handleSuccess) {
             handleSuccess(data, variables);
         }
     };
 
-    const settledFn = () => {
-        queryCache.invalidateQueries(DEFAULT_BAG_QUERY_KEY);
-
-        if (handleSettled) {
-            handleSettled();
-        }
-    };
-
-    const [mutate] = useMutation(updateDefaultBag, {
+    const { mutate } = useMutation(updateDefaultBag, {
         onSuccess: successFn,
         onError: handleError,
-        onSettled: settledFn,
```

Comment:
It turns out that `handleSettled` is being used by `analyses`, `apps`, and `data` listing components; but only with this `useBagAddItems` hook, in order to clear the current selection.

For example:
https://github.com/cyverse-de/sonora/blob/951c17d976ae588f106d001c7f953308c4810b98/src/components/data/listing/Listing.js#L517-L524